allennlp/data/vocabulary.py

```diff
@@ -210,20 +210,18 @@ def add_token_to_namespace(self, token: str, namespace: str='tokens') -> int:
             self._token_to_index[namespace][token] = index
             self._index_to_token[namespace][index] = token
             return index
-        else:
-            return self._token_to_index[namespace][token]
+        return self._token_to_index[namespace][token]
```

Comment:
I think I disagree with pylint on the readability of this particular error message, and I'm tempted to disable it.  It's not a strong opinion, though - what do you think?